robot/EDA/resources/EDA.robot

```diff
@@ -1,9 +1,46 @@
 *** Settings ***
 
-Resource       cumulusci/robotframework/Salesforce.robot
-Library        EDA.py
+Resource        cumulusci/robotframework/Salesforce.robot
+Library         robot/EDA/resources/EDA.py
+Library         DateTime
+
+*** Variables ***
+${PRINT_PACKAGE}        true
 
 *** Keywords ***
+Open Test Browser And Print Package Details
+    [Documentation]         Opens the test browser and runs a keyword that prints package details from setup
+    Open Test Browser
+    Run Keyword If          '${PRINT_PACKAGE.lower()}' == 'true'     Capture SAL and EDA package details
+
+
+Capture SAL and EDA package details
+    [Documentation]         Captures the package details from SAL setup page and prints it to console
+    Set Global Variable                 ${PRINT_PACKAGE}            false
+    Wait Until Loading Is Complete
+    Go To Setup Home
+    Wait For New Window                 Home | Salesforce
+    Select Window                       Home | Salesforce
```

Comment:
`Select Window` is deprecated. Use [Switch Window](http://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Switch%20Window) instead. Otherwise, when we switch to SeleniumLibrary 4.x (which is happening soon...) you'll get a warning or an error on this statement.

Reply:
@boakley - I think this is Safiya's code.  I believe she added this for detecting the package details.  Just a guess.  I'll rework this on my EDA rewrite - which is a different work item than this.

Reply:
I changed this to use 'Switch Window"